buildlib/az-distro-release.yml

```diff
@@ -31,6 +31,19 @@ jobs:
         ubuntu22_cuda11:
           build_container: ubuntu22_cuda11
           artifact_name: $(POSTFIX)-ubuntu22.04-mofed5-cuda11.tar.bz2
+        centos7_cuda12:
+          build_container: centos7_cuda12
+          artifact_name: $(POSTFIX)-centos7-mofed5-cuda12.tar.bz2
+        ubuntu18_cuda12:
```

Comment:
what about centos8_cuda12 ?

Reply:
CentOS 8 is retired and is no longer supported.
Do we still need it?

Reply:
CUDA 12 is only available for CentOS 7, not 8:
https://developer.nvidia.com/cuda-12-0-0-download-archive?target_os=Linux&target_arch=x86_64&Distribution=CentOS&target_version=7